.evergreen.yml

```diff
@@ -124,6 +124,23 @@ tasks:
 
         COVERALLS_REPO_TOKEN=${coveralls_repo_token|} poetry run coveralls
 
+- name: evg_api_install
```

Comment:
I think you have to add this to the list of tasks that will be executed in the build variant (top of the file, line 6)